kubernetes/provider.go

```diff
@@ -633,3 +634,26 @@ func useAdmissionregistrationV1beta1(conn *kubernetes.Clientset) (bool, error) {
 	useadmissionregistrationv1beta1 = ptrToBool(true)
 	return true, nil
 }
+
+func getServerVersion(connection *kubernetes.Clientset) (*gversion.Version, error) {
+	sv, err := connection.ServerVersion()
```

Comment:
We make use of this over here too: https://github.com/hashicorp/terraform-provider-kubernetes/blob/25512e4f8121192651ed5706e585a470cb73b699/kubernetes/resource_kubernetes_service.go#L445

Might be worth trying to unify the two.

Reply:
Thanks, made a small improvement. 👍🏻